src/Verifier/SAW/Conversion.hs

```diff
@@ -394,7 +394,7 @@ mkTupleSelector :: Int -> Term -> TermBuilder Term
 mkTupleSelector i t
   | i == 1 = mkTermF (FTermF (PairLeft t))
   | i > 1  = mkTermF (FTermF (PairRight t)) >>= mkTupleSelector (i - 1)
-  | otherwise = fail "mkTupleSelector: non-positive index"
+  | otherwise = error "mkTupleSelector: non-positive index"
```

Comment:
Why error instead of panic?

Reply:
That was a mistake, it should be `panic`. Actually I was thinking that we could get by without any error case at all in that function, by changing the argument type from `Int` to `Natural`, and making the tuple indexing 0-based to match Cryptol, but I think that's out of scope for this PR. I'll just change it to `panic`.